### PR TITLE
Add a couple of challenges for string section

### PR DESCRIPTION
**Pull request by ankitsmt211**
adds three challenges for `String` section

### CODE REVIEW

**Review by Taz03 — CHANGES_REQUESTED** [review]
fix formatting

**Comment by ankitsmt211** [comment]
@Taz03 how do i do that?

**Comment by Taz03** [comment]
if ur good enough then u should know how properly formatted java code looks likes, otherwise use ur ide and also get  better

**Comment by ankitsmt211** [comment]
@Taz03 i don't appreciate such comments, there's no need for it. There's no guidelines whatsoever anywhere in project, so i did my best.

**Comment by Taz03** [comment]
y u don't like it, i just said how to format ur code, i can even do it for u

**Comment by ankitsmt211** [comment]
>if you are good enough then you should know how to format your code

this
>get better

and this

there's no need for these comments, you could have simply either pointed me to a direction or what convention to follow or just ignored it and let someone else help.

File: src/strings/challenges.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+* Try not to give up until you've given it a solid attempt
+
+## challenge 1
+
+What will this program output when run? Write down your guess and then try running it.
+
+```Java
+public class Main {
+    public static void main(String[] args) {
+
```
Comment (Taz03) [inline src/strings/challenges.md]: remove this empty line

File: src/strings/challenges.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+```
+
+## challenge 2
+
+What will this program output when run? Write down your guess and then try running it.
+
+```Java
+public class Main {
+    public static void main(String[] args) {
+
```
Comment (Taz03) [inline src/strings/challenges.md]: .

File: src/strings/challenges.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+```
+
+## challenge 3
+
+What will this program output when run? Write down your guess and then try running it.
+
+```Java
+public class Main {
+    public static void main(String[] args) {
+
```
Comment (Taz03) [inline src/strings/challenges.md]: .

File: src/strings/challenges.md [hunk 4]
```diff
@@ -1 +1,59 @@
 # Challenges
+
+Remember the rules for this are
+
+* Try to use only the information given up to this point in this book.
+* Try not to give up until you've given it a solid attempt
+
+## challenge 1
```
Comment (Taz03) [inline src/strings/challenges.md]: first letter should be capital, no?

File: src/strings/challenges.md [hunk 5]
```diff
@@ -1 +1,59 @@
...
+        String second = "2";
+        String result = first + second;
+    
+        System.out.println(result);
+    }
+}
+
+```
+
+## challenge 2
```
Comment (Taz03) [inline src/strings/challenges.md]: .

File: src/strings/challenges.md [hunk 6]
```diff
@@ -1 +1,59 @@
...
+        String first = "1";
+        int second = 2;
+
+        System.out.println(first + second);
+    }
+}
+
+```
+
+## challenge 3
```
Comment (Taz03) [inline src/strings/challenges.md]: .

**Review by Taz03 — CHANGES_REQUESTED** [review]

**Comment by marko-radosavljevic** [comment]
@Taz03 Please be friendlier, you know we don't tolerate such attitudes. :)

Apologies for that @ankitsmt211, and thank you for your contribution. :relaxed: :heart:

**Comment by marko-radosavljevic** [comment]
At some point we should align on the formatting, but it's not terribly important at this stage.
 
Default intellij / google style is fine, which is basically the formatting we have been using for tj-bot too.

**Comment by Taz03** [comment]
@ankitsmt211 sorry ankit, don't get better

**Comment by bowbahdoe** [comment]
@ankitsmt211 I am going to integrate these on another branch.

**Comment by bowbahdoe** [comment]
@ankitsmt211 Integrated. Added you as co-author for that PR

**Pull request closed without merging**